lib/widget/helpers/widgets.dart

```diff
@@ -54,8 +54,12 @@ class BoxWrapper extends StatelessWidget {
               color: boxController.backgroundColor,
               image: boxController.backgroundImage?.image,
               gradient: boxController.backgroundGradient,
-              border: !boxController.hasBorder()
-                  ? null
+              border: boxController.borderGradient !=null
```

Comment:
Please make sure to test this next time. This will cause all border to show up on everything everywhere. Do something like:

if !hasBorder() => none
else if hasGradient => use GradientBorder
else use Border.all()